content/2_Bronze/Time_Comp.mdx

```diff
@@ -451,3 +452,74 @@ triplets.
 <!-- Bitsets are significantly faster (and so are bitwise operations vs iterating through bits). I don't want to make it sound insignificant -->
 
 For now, don't worry about optimizing constant factors -- just be aware of them.
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    What's time complexity?
+    <Quiz.Answer>
+      Measuring the amount of memory an algorithm consumes.
+      <Quiz.Explanation>
+        This is known as space complexity.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Measuring the number of operations an algorithm performs.
+      <Quiz.Explanation>
+        You'd always want your algorithm to run as fast as possible!

```

Comment:
Is there any indicator of a correct answer? If not, maybe change it to "Correct. You'd always want your algorithm..."